djangoproject/templates/diversity/changes.html

```diff
@@ -1,39 +1,51 @@
-    the same way we track changes to code. All changes will proposed via a pull request
-    to the
-    <a href="https://github.com/django/djangoproject.com">djangoproject.com repository
-      on GitHub</a>. Changes will be reviewed by the membership first, and then
-    sent to the DSF, the Django core team, and the Django community for comment.
-    We'll hold a comment period of at least one week,  and then each group will vote
-    on the change using its normal process (a board for the DSF,
-    <a href="{% url 'document-detail' lang='en' version='dev' url='internals/contributing/bugs-and-features' host 'docs' %}#how-we-make-decisions">
-      a core dev vote</a> for the core team).
-    Approved changes will be merged, published, and noted below.</p>
-
-  <p>This only applies to material changes; changes that don't effect the intent
-    (typo fixes, re-wordings, etc.) can be made immediately.</p>
-
-  <p>A complete list of changes can always be found
-    <a href="https://github.com/django/djangoproject.com/commits/main/djangoproject/templates/diversity">on GitHub</a>;
-    major changes and releases are summarized below.</p>
-
-  <h2>Changelog</h2>
+  <h1>{% translate "Django Community Diversity Statement - Changes" %}</h1>
+
+  <h2>{% translate "Change control process" %}</h2>
+  {% url 'document-detail' lang='en' version='dev' url='internals/contributing/bugs-and-features' host 'docs' as bugs_url %}
+  <p>
+    {% blocktranslate trimmed %}
+      We're (mostly) programmers, so we'll track changes to the diversity statement
+      the same way we track changes to code. All changes will proposed via a pull request
+      to the
+      <a href="https://github.com/django/djangoproject.com">djangoproject.com repository
+        on GitHub</a>. Changes will be reviewed by the membership first, and then
+      sent to the DSF, the Django core team, and the Django community for comment.
+      We'll hold a comment period of at least one week,  and then each group will vote
+      on the change using its normal process (a board for the DSF,
+      <a href="{{ bugs_url }}#how-we-make-decisions">
+        a core dev vote</a> for the core team).
+      Approved changes will be merged, published, and noted below.
+    {% endblocktranslate %}
+  </p>
+
+  <p>
+    {% blocktranslate trimmed %}
+      This only applies to material changes; changes that don't effect the intent
+      (typo fixes, re-wordings, etc.) can be made immediately.
+    {% endblocktranslate %}
+  </p>
+
+  <p>
+    {% blocktranslate trimmed %}
+      A complete list of changes can always be found
+      <a href="https://github.com/django/djangoproject.com/commits/main/djangoproject/templates/diversity">on GitHub</a>;
+      major changes and releases are summarized below.
+    {% endblocktranslate %}
+  </p>
+
+  <h2>{% translate "Changelog" %}</h2>
 
   <dl>
-    <dt>Jun 16, 2015</dt>
+    <dt>{% translate "June" %} 16, 2015</dt>
```

Comment:
This won't work either unfortunately (in french for example, the day comes before the month name).

I haven't tested it, but I think a custom tag like this should work:
```
from datetime import date

from django import template
from django.template.defaultfilters import date as datefilter

register = template.Library()


@register.simple_tag
def isodate(datestr, dateformat="DATE_FORMAT"):
    """
    Convert the given string to a date object (ISO) then format it using
    the given format (using Django's |date template filter)
    """
    d = date.fromisoformat(datestr)
    return datefilter(d, dateformat)
```
Then you can use it like so:
```suggestion
    <dt>{% isodate "2015-06-16" %}</dt>
```

Reply:
Ooo clever. I'll try that out. I was going for the "people will make sense of this" approach rather than thinking of a solution!

I'll try this out.